main/src/components/ActionBar/ActionBar.jsx

```diff
@@ -108,9 +115,8 @@ function ActionBar() {
                     <TableCell>
                       <SourceFetchStatusIcon source={source} />
                     </TableCell>
-                    <TableCell>
-                      <SourceVerificationIcon httpProxies={query.httpProxies} source={source} />
-                    </TableCell>
+                    { !!vc.enabled && <TableCell> <SourceVerificationIcon httpProxies={query.httpProxies} source={source} /> </TableCell> }
+                    { !!onchain.enabled &&<TableCell> <ChainVerificationIcon endpoint={onchain.endpoint} source={source} /> </TableCell> }
```

Comment:
I think this won't work if onchain.enabled = true and onchain.endpoint is not filled in